service-map/src/main/java/org/oskari/maplayer/model/MapLayerAdminInput.java

```diff
@@ -0,0 +1,18 @@
+package org.oskari.maplayer.model;
+
+import java.util.Map;
+
+/**
+ * Used as input data model for layers in the admin UI.
+ */
+public class MapLayerAdminInput extends MapLayer {
```

Comment:
Maybe we should refer to this as MapLayerVector instead of MapLayerAdminInput as all of these are more of an extension for vector layers than input from MapLayerAdmin? Or should we just add these to the MapLayer class directly? These are used for initializing layers from migrations as well.